nifi-assembly/pom.xml

```diff
@@ -721,7 +721,12 @@ language governing permissions and limitations under the License. -->
             <version>1.10.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
-
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-easyrules-nar</artifactId>
```

Comment:
not clear on size of this nar but generally we cannot add nars....unless really small

Reply:
good point @joewitt I will provide a profile to include this nar only if enabled for build.